meerk40t/core/node/op_image.py

```diff
@@ -37,10 +40,52 @@ def __init__(self, *args, **kwargs):
         self.allowed_elements_dnd = ("elem image",)
         # Which elements do we consider for automatic classification?
         self.allowed_elements = ("elem image",)
+        self.stopop = True
+        self.allowed_attributes = []
 
     def __repr__(self):
         return "ImageOpNode()"
 
+    def __str__(self):
```

Comment:
The str stuff is added in this PR because it's not actually in main anymore. It defaults to the `node.__str__()` call which uses the default map and `self._formatter` (which gets set prior to checking labels) so the nodes str() and label remain the same. And will have the formatter overwritten on device switch.

Reply:
Done